### PR TITLE
Initial Update to the Documentation

### PR DESCRIPTION
**Pull request by caffix**
Starting updates to the documentation

The documentation is written to reflect the new 3.0.0 interface

### CODE REVIEW

File: doc/install.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+Details regarding this package can be found [here](https://github.com/BlackArch/blackarch/blob/master/packages/amass/PKGBUILD)
+
+### FreeBSD
+
+```bash
+cd /usr/ports/dns/amass/ && make install clean
+pkg install amass
+```
+
```
Comment (kalbasit) [inline doc/install.md]:
```suggestion

### Nix or NixOS

```bash
nix-env -f '<nixpkgs>' -iA amass
```

I can't embed code block inside a suggestion block, but you get the idea :)

Comment (caffix) [reply doc/install.md]: Excellent! Thank you, @kalbasit

File: doc/user_guide.md [hunk 2]
```diff
@@ -1,133 +1,255 @@
...
-```bash
-amass -src -ip --active -brute -do owasp.json -d owasp.org
-```
+| Flag | Description | Example |
+|------|-------------|---------|
+| -config | Path to the INI configuration file | amass db -config config.ini |
+| -d | Domain names separated by commas (can be used multiple times) | amass db -d example.com |
+| -df | Path to a file providing root domain names | amass db -df domains.txt |
+| -dir | Path to the directory containing the graph database | amass db -dir PATH |
+| -i | Path to the Amass data operations JSON input file | amass db -i PATH |
```
Comment (fork-while-fork) [inline doc/user_guide.md]: Can you update this to match the changes to the command line tool?

**Comment by caffix** [comment]
Thank you everyone

**Pull request merged**